specification/redis/resource-manager/Microsoft.Cache/stable/2021-06-01/redis.json

```diff
@@ -2693,6 +2743,20 @@
         }
       },
       "description": "The response of listUpgradeNotifications."
+    },
+    "OperationStatusResult": {
```

Comment:
could you rename it , as it conflicts with the common types and caused circular reference

Reply:
ok, sure.